### ISSUE TITLE
<For> may duplicate items.

### ISSUE THREAD

**Issue by godstail** [leptos-rs/leptos#1279]
**Describe the bug**
In a very simple case, items (single number in item view) get duplicated. Adding {" "} to item view removes duplication.

**Leptos Dependencies**
```toml
leptos = { git = "https://github.com/leptos-rs/leptos", features = ["serde", "nightly"] }
leptos_axum = { git = "https://github.com/leptos-rs/leptos", optional = true }
leptos_meta = { git = "https://github.com/leptos-rs/leptos", features = ["nightly"]}
leptos_router = { git = "https://github.com/leptos-rs/leptos", features = ["nightly"] }
```

**To Reproduce**
```rust
#[component]
pub fn TestComp(
    cx: Scope,
) -> impl IntoView {
    let (vector, _) = create_signal(cx, vec![0, 1]);
    view! {cx, 
        <For
            each=vector
            key=|item| *item
            view=move |_cx, item| view!{cx, 
                {item}
                // {" "} // adding this cures duplication
            }
        />
    }
}
```
**Expected behavior**
no duplication

**Comment by godstail** [comment]
Yes. Enclosing item in a div also fixes the behaviour.

**Comment by gbj** [comment]
This should be fixed by #1281. Just FWIW you can simplify a bunch:
```rust
view=move |_cx, item| view!{cx, 
    {item}
}
```
is the same as 
```rust
view=move |_cx, item| item
```
